### ISSUE TITLE
Tool to check whether the site is accessible or not

### ISSUE THREAD

**Issue by venkat2305** [LagradOst/CloudStream-3#915]
Can you add an option to check for the websites which works on my network. My network doesn't allow me to change DNS , so instead of checking each and every site manually , can you enable an option that reduces this friction.

**Comment by LagradOst** [comment]
There is a built in dns option, check that.

**Comment by venkat2305** [comment]
Yeah , but when i changed the dns (in the app itself) , my network goes down.
